docker/docker-compose.yml

```diff
@@ -0,0 +1,108 @@
+version: '3.8'
+
+services:
+
+  #------- Secrets
+  generator:
+    image: 792926601177.dkr.ecr.us-east-2.amazonaws.com/specular-platform:specular-latest
+    volumes:
+      - /opt/specular/specular/workspace:/specular/workspace
+      # - sp_workspace:/specular/workspace
+    working_dir: /specular/workspace
+    entrypoint: /bin/bash ../sbin/generate_secrets.sh -d -j -y
+
+  #------- L1
+  l1-geth:
+    image: 792926601177.dkr.ecr.us-east-2.amazonaws.com/specular-platform:specular-latest
+    tty: true
+    healthcheck:
+      test: bash -c "[ -f /tmp/.l1_started.lock ]"
+      retries: 10
+      interval: 5s
+      start_period: 40s
+    volumes:
+      - /opt/specular/specular/workspace:/specular/workspace
+    working_dir: /specular/workspace
+    entrypoint: /bin/bash ../sbin/start_l1.sh -c -d -w -y
+    depends_on:
+      generator:
+        condition: service_completed_successfully
+    networks:
+      specular:
+        ipv4_address: 192.168.160.2
+    ports:
+      - "8545:8545"
+
+
+  #------- GETH
+  sp-geth:
+    image: 792926601177.dkr.ecr.us-east-2.amazonaws.com/specular-platform:specular-latest
+    volumes:
+      - /opt/specular/specular/workspace:/specular/workspace
+    restart: on-failure
+    depends_on:
+      l1-geth:
+        condition: service_healthy
+    healthcheck:
+      test: bash -c "[ -f /tmp/.sp_geth_started.lock ]"
+      retries: 10
+      interval: 5s
+      start_period: 20s
```

Comment:
I remember we set this at `40s` on our call yesterday, is `20s` working? No worries if this works.